Jenkinsfile

```diff
@@ -155,10 +171,23 @@ builds.ath = {
       // Just to be safe
       deleteDir()
       checkout scm
+      def browser = 'firefox'
```

Comment:
Pulling out the browser into a local variable so I can pass it both to ATH and to `launchable record tests` as the [flavor](https://www.launchableinc.com/docs/sending-data-to-launchable/using-the-launchable-cli/recording-test-results-with-the-launchable-cli/use-flavors-to-run-the-best-tests-for-an-environment/), along with platform (Linux or Windows) and JDK version. It is good to start sending this information to Launchable it allows Launchable to produce better subsets in the future. For example, when producing a subset of tests for Windows test runs it will try to pick tests that are more relevant to Windows builds. The idea here is that if we ever change the matrix to include more browsers, the Launchable plumbing for browser flavor would already be taken care of.